File: programmers/2021_웹백엔드개발자_상반기/로또의최고순위와최저순위.py
def solution(lottos, win_nums):
    answer = []
    cnt=0
    num=0
    score=[6,6,5,4,3,2,1]
    for i in lottos:
        if i==0:
            cnt+=1
        else:
            if i in win_nums:
                num+=1
    answer.append(score[cnt+num])
    answer.append(score[num])
    return answer

File: javascript/programmers/Level1/로또의최고순위와최저순위.js
function solution(lottos, win_nums) {
  var answer = [];
  var result = [6, 6, 5, 4, 3, 2, 1];
  var cnt = 0;
  var same = 0;
  lottos.forEach((lotto) => {
    if (lotto === 0) cnt++;
    else {
      if (win_nums.includes(lotto)) {
        same++;
      }
    }
  });
  answer.push(result[cnt + same]);
  answer.push(result[same]);
  return answer;
}
